webpages/settings/components/addon-setting.html

```diff
@@ -73,15 +73,18 @@
         :disabled="!addon._enabled"
       />
       <div v-if="setting.type === 'select'" class="filter-options">
-        <button
-          class="filter-option"
-          v-for="option of setting.potentialValues"
-          :class="{'sel' : addonSettings[setting.id] === option.id, 'disabled': !addon._enabled}"
-          :disabled="!addon._enabled"
-          @click="updateOption(option.id);"
-        >
-          {{ option.name }}
-        </button>
+        <div v-for="option of setting.potentialValues">
```

Comment:
Can this outer div be a `template` instead? That would fix the label's `flex-grow` style not being applied, causing the options to look like this when stacked vertically (in the popup):

<img width="432" height="90" alt="image" src="https://github.com/user-attachments/assets/6c886650-bca8-4371-8707-479004b758ad" />